posts/project-mycorrhiza/index.md

```diff
@@ -0,0 +1,188 @@
+Photo by [Malene Thyssen](https://commons.wikimedia.org/wiki/User:Malene) - Own work, [CC BY-SA 3.0](http://creativecommons.org/licenses/by-sa/3.0/ "Creative Commons Attribution-Share Alike 3.0"), [Link](https://commons.wikimedia.org/w/index.php?curid=752225)
+
+<!--truncate-->
+
+---
+
+[TL;DR at the bottom!](#tldr)
+
+Before I explain what Project Mycorrhiza is and what it's about, lets talk about
+the Packit dashboard and why it's been getting so much love from me the past year.
+
+It's to no surprise that Packit is used within a lot of different projects around
+Fedora, which includes [Convert2RHEL](https://github.com/oamg/convert2rhel) that I'm
+a developer of. But I've always felt like Packit's dashboard has missed the mark on
+being the go-to for getting an overview for the pipeline, results, or even the
+project itself.
+
+Since I like making websites better for my own benefit, I've spending a huge amount of my own
+time the past year to gradually improve the Packit dashboard to prepare it for something
+bigger. Which is where Project Mycorrhiza comes in.
+
+## What's mycorrhiza and mycorrhizal network?
+
+:::note
+
+This is heavily simplified for the purpose of getting a quick overview. I encourage you
+to read more about [mycorrhiza on the wikipedia article about it](https://en.wikipedia.org/wiki/Mycorrhiza)
+and any related articles!
+
+:::
+
+You might be familiar with something called mycelium, the network of root-like structures that
+fungi form. I'm assuming most of you have seen this form on food before, such as bread
+that grew mold.
+
+The mycelia helps fungi get nutrients and get enough food to grow mushrooms!
+Some are just terrible to be around, some are interesting to look at but relatively toxic, and
+some are even edible. Those same labels I applied to the mushrooms are something plants and trees
+sort of do as well!
+
+A mycorrhizal network is a symbiosis between fungi and flowers and trees.
+You have already seen one too as the photo above is a bunch of Beech trees that
+are "[ectomyccorhizal](https://en.wikipedia.org/wiki/Ectomycorrhiza)"!
+What it means is that nutriets are shared between the two, the fungi spreads out mycelia in the soil
+and transfers nutrients to the tree, and the tree provides sugar and fat back to the fungi.
+
+Since my project relates to integrating more services and getting a better
+packaging ecosystem, I chose the name Project Mycorrhiza to put emphasis on symbiosis between
+the different tools.
+
+## Project Mycorrhiza
+
+Now with the name explanation out of the way. First and foremost, this should be seen as a kind
+of idea or kind of a temporary working group to see about the future goals of Packit and
+how the Packit dashboard fits into the Fedora service ecosystem. The Mycorrhiza name itself
+will only stick around for the duration of this investigation and alignment. Once it is done
+we will likely drop or change it.
+
+### Fedora packaging ecosystem
```

Comment:
this section focuses on the Copr builds and tests, do we want to also mention the downstream release automation here or you would say that's out of scope for now?

Reply:
I don't fully understand how the downstream release automation works so I didn't want to bring it up and be incorrect